docs/source/api/deepchecks.nlp.checks.rst

```diff
@@ -0,0 +1,15 @@
+checks
+======
+
+.. automodule:: deepchecks.nlp.checks
+
+.. currentmodule:: deepchecks.nlp.checks
+
+.. rubric:: Modules
+
+.. autosummary::
+    :toctree: generated
+    :template: autosummary/check-module.rst
+    
+    train_test_validation
+    model_evaluation
```

Comment:
data_integrity?

Reply:
Nice catch, though I think @noamzbr fixes that whole thing anyway in his next PR.

Reply:
yep fixed in mine